### PR TITLE
refactor: :label: Migrated component to typescript

### PR DESCRIPTION
**Pull request by arshad-rao-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.

### CODE REVIEW

File: packages/cfd/src/Components/cfd-real-account-display.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ type TCurrentList = DetailsOfEachMT5Loginid & {
     enabled: number;
 };
 
-type TCFDRealAccountDisplayProps = {
+export type TCFDRealAccountDisplayProps = {
```
Comment (farzin-deriv) [inline packages/cfd/src/Components/cfd-real-account-display.tsx]:
```suggestion
type TCFDRealAccountDisplayProps = {
```
@arshad-rao-deriv Prop types of a component are scoped to the component itself, Ideally we don't need to export them. If you need the type somewhere else (test files) you can use `React.ComponentProps<typeof ComponentName>` generic to get the type 🙇🏻

Comment (arshad-rao-deriv) [reply packages/cfd/src/Components/cfd-real-account-display.tsx]: Updated.

**Review by hamza-deriv — APPROVED** [review]

File: packages/cfd/src/Components/__tests__/cfd-download-container.spec.tsx [hunk 2]
```diff
@@ -10,20 +10,34 @@ jest.mock('@deriv/components', () => {
...
+        derivez_tokens: {
+            demo: '',
+            real: '',
+        },
     };
 
     it('should render <CFDDownloadContainer /> correctly', () => {
         render(<CFDDownloadContainer {...mock_props} />);
         expect(screen.getByTestId(/dt_cfd_dashboard_download_center_container/i)).toBeInTheDocument();
+        // await waitFor(() => {
```
Comment (mahdiyeh-deriv) [inline packages/cfd/src/Components/__tests__/cfd-download-container.spec.tsx]: @arshad-rao-deriv remove this comment

File: packages/cfd/src/Components/__tests__/cfd-poi.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        onSave: jest.fn(),
+        onSubmit: jest.fn(),
+        addNotificationMessageByKey: jest.fn(),
+        removeNotificationByKey: jest.fn(),
+        removeNotificationMessage: jest.fn(),
+        jurisdiction_selected_shortcode: 'svg',
+        value: {
+            poi_state: 'unknown',
+        },
+    };
```
Comment (mahdiyeh-deriv) [inline packages/cfd/src/Components/__tests__/cfd-poi.spec.tsx]:
```suggestion
    } as const;
```

File: packages/cfd/src/Components/__tests__/cfd-download-container.spec.tsx (hunk 2 above)
Comment (arshad-rao-deriv) [reply packages/cfd/src/Components/__tests__/cfd-download-container.spec.tsx]: Updated.

File: packages/cfd/src/Components/__tests__/cfd-poi.spec.tsx (hunk 3 above)
Comment (arshad-rao-deriv) [reply packages/cfd/src/Components/__tests__/cfd-poi.spec.tsx]: Updated.

**Pull request merged**